taxonomies/categories.txt

```diff
@@ -5415,7 +5415,7 @@ nl:Heiligenbloem
 wikidata:en:Q1543811
 
 <en:Herbal teas
-en:Dandelion, Common dandelion, Raw dandelion
+en:Dandelion, Common dandelion
```

Comment:
not related to your change, but this should not be under herbal teas, or we should rename it to "dandelion herbal teas" (or make a separate entry)

Reply:
``` 
<en:Herbal teas
en:Dandelion herbal tea

<en:Herbs
en:Dandelion, Common dandelion
```